src/Value/Component.php

```diff
@@ -0,0 +1,11 @@
+<?php
+
+namespace Sabberworm\CSS\Value;
+
+/**
+ * To be used by any class that represents a component of a CSS property value.
+ * Values often comprise 'component operator component' (recursively).
+ * This interface has no methods to implement;
+ * its purpose is abstract: to allow a unique type to be specified when a `Component` is an argument of a method.
+ */
+interface Component {}
```

Comment:
We also should make sure that the name makes sense globally (as Mermaid on GitHub doesn't allow namespaces). Maybe `CssPropertyValueComponent` oder `PropertyValueComponent`?

Reply:
> We also should make sure that the name makes sense globally (as Mermaid on GitHub doesn't allow namespaces). Maybe `CssPropertyValueComponent` oder `PropertyValueComponent`?

I don't think whatever limitations of Mermaid there may be should dictate class naming conventions.  Mermaid might in future be enhanced to show the namespaces.  We should choose the most apt names for classes, interfaces and namespaces independently of whatever other tools we use.

The FQN is `Value\Component`.  The namespace `Value` could perhaps be `PropertyValue` (or `Property\Value`) for clarity, though any such rename would be beyond the scope of this PR, and probably unnecessary (it would be a breaking change).